server/utility/minerva_utility.py

```diff
@@ -48,6 +48,11 @@ def findMinervaFolder(currentUser, user, create=False):
                            PluginSettings.MINERVA_FOLDER, create)
 
 
+def findPublicFolder(currentUser, user, create=False):
```

Comment:
My editor was changing the file modes and it also happened here. Can you change the file mode back to 100644? Or I can do it.